.env

```diff
@@ -19,6 +19,9 @@ SUPPORT_NO_CATEGORY_VIDEOS=true
 SUPPORT_NEW_CATEGORIES=true
 KILL_SWITCH_ON=false
 VIDEO_VIEW_PER_IP_TIME_LIMIT=86400 # 86400 seconds = 24 hours
+VIDEO_RELEVANCE_VIEWS_TICK=50 # every 50 views video relevance score will be recalculated
+RELEVANCE_WEIGHTS="[1, 0.03, 0.3, 0.5]" # [newness (30 days - days since created) weight, views weight, comments weight, rections weights]
```

Comment:
Are these the same weights as:
```
Y = A*(30-days since published) +Bviews + Cnumber of comments + D**number of reactions

To start with:
A= 0.4
B= 0.1
C = 0.3
D= 0.2
```
?